jujuzitroni/plants-need-water
# calendar

---now with fixed bugs!----

in this PR I added the calendar with its functionality.

find the calendar under the 'calendar' tab.

https://plants-need-calendarcom-8tadap.herokuapp.com/calendar

to test it:
- add a habit
- check the habit
- see the habit color as a dot on the current day in calendar
- you can also click a day in the calendar to go to the habit list of that day and check a habit for that day.

I am aware of two bugs in this PR:
- if you click a day in the calendar you land on the list for the day before. 
- if you click a day in the calendar the 'today' tab isn't active but it should be.
In both cases I have an idea what the issue is and will look into it (but wanted to post the PR tonight anyways).


fixes #10
